meta-ibm/meta-witherspoon/recipes-phosphor/vpd/mihawk-openpower-fru-vpd-layout/layout.yaml

```diff
@@ -4,6 +4,7 @@ BMC:
         OPFR,VS: SerialNumber
         OPFR,VN: Manufacturer
         VNDR,IN: Model
+        VINI,B3: SubModel
```

Comment:
I wonder why we chose "SubModel", surely the B3 keyword has nothing to indicate the part's model?

Reply:
Because these names can only be used in [Asset.interface](https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/OP940/xyz/openbmc_project/Inventory/Decorator/Asset.interface.yaml), it seems even more inappropriate.
So I [added a new name SubModel](https://github.com/ibm-openbmc/phosphor-dbus-interfaces/pull/15) to use (same as the name of the [upstream commit](https://github.com/openbmc/phosphor-dbus-interfaces/commit/ff4702aa2163f16822a3500b3986fa24571cea95)).

If there are any misunderstandings or better suggestions, please let me know, thank you